web/pgadmin/browser/server_groups/servers/databases/schemas/statistics/tests/utils.py

```
@@ -0,0 +1,392 @@
+        query = (
+            f"CREATE STATISTICS {schema_name}.{statistics_name} "
+            f"({stat_types_str}) ON {exprs_str} "
+            f"FROM {schema_name}.{table_name}"
+        )
+        pg_cursor.execute(query)
+
+        pg_cursor.execute(
+            f"SELECT s.oid FROM pg_catalog.pg_statistic_ext s "
+            f"JOIN pg_catalog.pg_namespace n ON s.stxnamespace = n.oid "
+            f"WHERE s.stxname = '{statistics_name}' "
+            f"AND n.nspname = '{schema_name}'"
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)